oneflow/core/functional/impl/nn_functor.cpp

```diff
@@ -3066,6 +3066,58 @@ class FusedBiasAddDropoutFunctor {
   std::shared_ptr<OpExpr> fused_bias_add_mask_scale_op_;
 };
 
+class FusedGluFunctor {
+ public:
+  FusedGluFunctor() {
+    op_ = CHECK_JUST(one::OpBuilder("fused_glu")
+                         .Input("x")
+                         .Input("w")
+                         .Input("b")
+                         .Input("v")
+                         .Input("c")
+                         .Output("y")
+                         .Output("matmul_wx")
+                         .Output("matmul_vx")
+                         .Build());
+  }
+
+  Maybe<Tensor> operator()(const std::shared_ptr<one::Tensor>& x,
+                           const std::shared_ptr<one::Tensor>& w,
+                           const std::shared_ptr<one::Tensor>& b,
+                           const std::shared_ptr<one::Tensor>& v,
+                           const std::shared_ptr<one::Tensor>& c,
```

Comment:
```suggestion
                           const Optional<one::Tensor>& v,
                           const Optional<one::Tensor>& c,
```

op_ 这个变量要根据是否有 vc 写两个，可以参考这个文件中其他有Optional输入的 functor